docs/installation-openshift-4.x/jdkhttp/README.md

```diff
@@ -0,0 +1,113 @@
+#### Login into Openshift cluster
+```sh
+oc login -u system:admin
+eval $(minishift oc-env) 
```

Comment:
Minishift does not support OSE 4 as far as I know.

Reply:
Fixed